tests/client/core/reducers/test_installations.js

```diff
@@ -216,6 +217,32 @@ describe('installations reducer', () => {
       });
   });
 
+  it('updates the status on INSTALL_CANCELLED', () => {
+    const state = {
+      'my-addon@me.com': {
+        guid: 'my-addon@me.com',
+        url: 'https://cdn.amo/download/my-addon.xpi',
+        downloadProgress: 100,
+        status: DOWNLOAD_PROGRESS,
+      },
+    };
+    assert.deepEqual(
```

Comment:
The only important changes are `status` and `downloadProgress` so this would be more refactor-friendly as the following assertion:
````js
const newState = installations(state, {
  type: INSTALL_CANCELLED,
  payload: {
    guid: 'my-addon@me.com',
  },
});
assert.equal(newState['my-addon@me.com'].downloadProgress, 0);
assert.equal(newState['my-addon@me.com'].status, UNINSTALLED);
````